uint64 timestamp	# time since system start (microseconds)
uint64 timestamp_sample # the timestamp of the raw data (microseconds)
uint32 seq		# Capture sequence number

uint32 ORB_QUEUE_LENGTH = 2
